lib/cogctl/actions/groups/add.ex

```diff
@@ -25,13 +25,13 @@ defmodule Cogctl.Actions.Groups.Add do
   end
 
   defp do_add(_endpoint, _, :undefined) do
-    display_error("Email address is required")
+    display_error("User is required")
```

Comment:
Of course, this may suggest a broader conversation about exactly what terminology we use across _all_ our code, and not just this isolated error message.